app/src/main/java/org/p2p/wallet/utils/BundleExtensions.kt

```diff
@@ -0,0 +1,71 @@
+package org.p2p.wallet.utils
+
+import android.os.Bundle
+import android.os.Parcelable
+import java.io.Serializable
+
+fun <T : Parcelable> Bundle.requireParcelable(key: String): T {
+    return if (containsKey(key)) {
+        requireNotNull(getParcelable(key))
+    } else {
+        throw IllegalArgumentException("Required argument \"$key\" is missing!")
+    }
+}
+
+fun Bundle.requireString(key: String): String {
```

Comment:
i saw that the naming style for this type of extensions is like:

1. getOrThrow / getOrNull / getOrDefault
so i suggest to use getOrThrow